### ISSUE TITLE
Adding test files

### ISSUE THREAD

**Issue by TornaxO7** [Roughsketch/nds#4]
Hi! Your test functions use some `.nds` files, like `big.nds` as you can see here:
```rust
    #[test]
    fn checksum_matches() {
        assert!(Extractor::new("big.nds", true).is_ok());
        assert!(Extractor::new("small.nds", true).is_ok());
    }
```
But they are not in this repository. Please add them!

**Comment by TornaxO7** [comment]
@Roughsketch are you there? ;-;

**Comment by Roughsketch** [comment]
Commented on the other merge, but I do not know what those two files are and don't have them locally. TinyFS.nds can be found [here](http://imrannazar.com/content/files/TinyFB.nds). As mentioned in the PR, I'll just consider tests broken until new ones are found to be used. This project isn't exactly actively maintained so it isn't going to be top priority for me to add tests.

**Comment by TornaxO7** [comment]
> This project isn't exactly actively maintained so it isn't going to be top priority for me to add tests.

Hm... if you want, I can co-maintain this crate for the time being. Because I want to write a NDS-Emulator and I need to read from `.nds` files for that.

**Comment by Roughsketch** [comment]
I'm open to that. I have a lot of other stuff going on so cannot focus on it.

**Comment by TornaxO7** [comment]
My latest [pull request](https://github.com/Roughsketch/nds/pull/6) should fix this issue.

**Comment by Roughsketch** [comment]
I think this can be closed with the merge of #6
